### DISCUSSION TITLE
Duplication in Immich due to Google Photos vs Backup from Phone

### DISCUSSION THREAD

**Discussion by Rwarcards762** [immich-app/immich#3566] [Q&A] [answered]
![image](https://github.com/immich-app/immich/assets/37875168/6d4bcca7-956c-468a-a3c6-351f68521162)

Hi all! Immich seems to be wonderful thus far, however I do have one issue.

My first use was downloading all of my images from Google Photos to import, and now I've set my phone to backup and sync... however now it seems my entire phone is going to upload a *duplicate* of everything previously imported in Google Photos.

Is there a good way to clean up these duplicates, will Immich automatically clean them up, etc? The issue is I think Google Photos had compressed version while my phone is now uploading the non-compressed versions. They are identical otherwise.

Example:

![image](https://github.com/immich-app/immich/assets/37875168/90201a87-deee-4310-9527-04185e89590d)

![image](https://github.com/immich-app/immich/assets/37875168/79f2521d-aa0b-4ffe-b3dd-d245acb65c1f)

Thanks in advance!

I'm hoping the solution isn't just nuking it and *not* importing the Google Photos download I did, but if that's the case, I can do that... ugh.

**Comment by alextran1502** [comment, marked as answer]
Hello, Immich deduplication mechanism uses the content (hash) of the file, so if a file is compressed, the hash will be different. So unfortunately you will have to nuke the installation.

**Reply by Rwarcards762** [reply]
😔 Figures. Makes sense though. Thanks for the quick answer!

**Reply by agross** [reply]
@Rwarcards762 You might want have a look at this: https://github.com/agross/immich-duplicates
